requirements/development.txt

```diff
@@ -104,11 +104,6 @@ pylint==2.17.7
     # via -r requirements/development.in
 python-ldap==3.4.3
     # via -r requirements/development.in
-requests==2.31.0
-    # via
-    #   pydruid
-    #   tableschema
-    #   tabulator
```

Comment:
Since `requests` is already pulled in by `shillelagh` in `base.txt`, it's no longer explicitly needed in the development requirements.